clients/stellar-relay-lib/src/connection/services.rs

```diff
@@ -152,7 +152,7 @@ pub(crate) async fn receiving_service(
 	actions_sender: mpsc::Sender<ConnectorActions>,
 	timeout_in_secs: u64,
 	retries: u8,
-) -> Result<(), Error> {
```

Comment:
this function is called inside a `tokio::spawn()` that doesn't return anything. I find it best to log the errors immediately rather than propagating it up.

Reply:
Got it, makes sense 👍